todo_app/app.py

```diff
@@ -10,51 +10,45 @@ def create_app():
     @app.route('/')
     def index():
 
-        trello_list_ids = {'todo':get_trello_list_id('To Do'),'doing':get_trello_list_id('Doing'),'done':get_trello_list_id('Done')}
+        list_ids = {'todo':'todo','doing':'doing','done':'done'}
```

Comment:
Not a problem, but the fact these now all have matching keys & values suggests we could just throw this away!